definitions.py

```diff
@@ -11,22 +11,26 @@
 
 class LoggingLevels(IntEnum):
     INFO = logging.INFO
+    DEBUG = logging.DEBUG
     WARNING = logging.WARNING
     ERROR = logging.ERROR
     CRITICAL = logging.CRITICAL
     FATAL = logging.FATAL
     OFF = 100
 
 
-# Default is full logging.
-DEFAULT_LOGGING_LEVEL = LoggingLevels.INFO
+# Default logging from LOG_LEVEL env variable otherwise INFO
+try:
+    LOG_LEVEL = os.environ['LOG_LEVEL'].upper()
+except KeyError:
+    LOG_LEVEL = 'INFO'
+
+DEFAULT_LOGGING_LEVEL = LoggingLevels[LOG_LEVEL]
```

Comment:
LOG_LEVEL environment variable now can be set to one of the possible values listed in LoggingLevels class `INFO`, `DEBUG`, `WARNING`, `ERROR`, `CRITICAL` or `FATAL`.
If LOG_LEVEL env var is not set, `INFO` will be used as default.